pubsub/tests/unit/pubsub_v1/subscriber/test_policy_thread.py

```diff
@@ -194,6 +194,7 @@ def test_on_response():
     # Set up the policy.
     policy = create_policy(executor=executor)
     policy._callback = callback
+    policy.modify_ack_deadline = mock.Mock()
```

Comment:
I would use `patch.object` with autospeccing.

Reply:
@jonparrott Does that work better?